Firmware/ultralcd.cpp

```diff
@@ -6842,6 +6879,7 @@ void lcd_print_stop()
           SERIAL_ECHOLNPGM("// action:cancel");   // for Octoprint
```

Comment:
isn't this used at other places too? It would be nice to extract the "// action:cancel" string and reuse it -> save FLASH ;)
I know it is not your fault, but when we are improving the code around...
just like you did MSG_OCTOPRINT_PAUSED